### PR TITLE
`Development`: Update Orion documentation

### PR DESCRIPTION
**Pull request by Kroko-fant**
@coderabbitai ignore

#### General
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).

### Motivation and Context

The documentation for Orion was not up to date. 

### Description
The documentation got enhanced with new features and changed screenshots of UI elements.

### Steps for Testing
Look at the built documentation at  and check that it looks like before.

#### Manual Tests
- [x] Test 1
- [ ] Test 2

### CODE REVIEW

**Comment by pzdr7** [comment]
Link: https://artemis-platform--8328.org.readthedocs.build/en/8328/user/orion.html

Looks good 👍 

In a few places, the documentation still refers to IntelliJ (rather than the more generic "IDE" / "all JetBrains IDEs"). Can you double-check if that's correct?

**Comment by Kroko-fant** [comment]
> Link: https://artemis-platform--8328.org.readthedocs.build/en/8328/user/orion.html
> 
> Looks good 👍
> 
> In a few places, the documentation still refers to IntelliJ (rather than the more generic "IDE" / "all JetBrains IDEs"). Can you double-check if that's correct?

Good Point I updated the text

File: docs/user/orion.rst [hunk 1]
```diff
@@ -10,12 +10,12 @@ Orion
 Overview
 --------
 
-The `Orion plugin <https://github.com/ls1intum/Orion>`_ for `IntelliJ <https://www.jetbrains.com/idea/>`_ supports students, teaching assistants (tutors), and instructors with the conduction of programming exercises. It integrates Artemis in IntelliJ and automates the download (clone) and upload (push) of exercises.
+The `Orion plugin <https://github.com/ls1intum/Orion>`_ for `IntelliJ <https://www.jetbrains.com/idea/>`_ supports students, teaching assistants (tutors), and instructors with the conduction of programming exercises. It integrates Artemis in all JetBrains IDEs and automates the download (clone) and upload (push) of exercises.
```
Comment (pzdr7) [inline docs/user/orion.rst:13]: One last reference to IntelliJ

**Review by pzdr7 — APPROVED** [review]
LGTM. There's one more reference to IntelliJ that I've highlighted

**Review by b-fein — DISMISSED** [review]

**Review by b-fein — APPROVED** [review]

**Review by pzdr7 — APPROVED** [review]

**Pull request merged**